src/read.c

```diff
@@ -3081,6 +3081,11 @@ avifResult avifDecoderParse(avifDecoder * decoder)
             // probably exif or some other data
             continue;
         }
+        if (item->auxForID) {
+            // NON-STANDARD: Allow subimages to not have an ispe property
```

Comment:
Nit: Change "subimages" to "auxiliary images"?

The reason is that we also refer to the primary image item as the "color subimage" in some comments and messages.

Reply:
I've updated this comment to use `auxiliary images`.